config.php

```diff
@@ -164,4 +164,8 @@
 $THEME->yuicssmodules = array();
 $THEME->rendererfactory = 'theme_overridden_renderer_factory';
 $THEME->requiredblocks = ' ';
-$THEME->addblockposition = BLOCK_ADDBLOCK_POSITION_DEFAULT;
+if (get_config('theme_boost_campus', 'settingsaddablockposition') == 'no') {
```

Comment:
Please change this according the new setting variable values (see comment in settings.php file)